CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the INCORE documents generated by Sphinx package will be
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased] - 2023-12-12
```

Comment:
sorry for nitpicking. For Unrelease we don't need to add a date yet.
I will approve assuming you have already removed it :-)